### ISSUE TITLE
Applications or Patches do not appear in the Manager

### ISSUE THREAD

**Issue by LLuanZZ** [ReVanced/revanced-manager#1641]
### Bug description

No applications appear when I go to "Select an application" nor do patches appear when I choose the YouTube APK.
![Screenshot_20240116-101128](https://github.com/ReVanced/revanced-manager/assets/108953018/d99a5f53-34b8-4e17-a7b3-60511474fe36)

Sorry for my English, I'm using Google Translate

### Version of ReVanced Manager and version & name of application you tried to patch

Revanced Manager: 1.18.0
Youtube 18.45.41 and 18.45.43

### Installation type

None

### Device logs

```shell
--------- beginning of main
01-16 10:33:35.491 32305 32305 E SurfaceSyncer: Failed to find sync for id=0
01-16 10:33:35.491 32305 32305 E SurfaceSyncer: Failed to find sync for id=1
01-16 10:33:35.491 32305 32305 E SurfaceSyncer: Failed to find sync for id=2
01-16 10:33:35.491 32305 32305 E SurfaceSyncer: Failed to find sync for id=3
01-16 10:33:35.491 32305 32305 E SurfaceSyncer: Failed to find sync for id=4
01-16 10:33:35.491 32305 32305 E SurfaceSyncer: Failed to find sync for id=5
01-16 10:33:35.491 32305 32305 E SurfaceSyncer: Failed to find sync for id=6
01-16 10:33:35.491 32305 32305 E SurfaceSyncer: Failed to find sync for id=7
01-16 10:33:35.491 32305 32305 E SurfaceSyncer: Failed to find sync for id=8
```

### Acknowledgements

- [X] This request is not a duplicate of an existing issue.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.
- [X] The issue is solely related to the ReVanced Manager

**Comment by oSumAtrIX** [comment]
What is the current API set to?

**Comment by LLuanZZ** [comment]
> Para que é definida a API atual?

![Uploading Screenshot_20240116-114404.png…]()

**Comment by LLuanZZ** [comment]
> What is the current API set to?

![Uploading Screenshot_20240116-114404.png…]()

**Comment by LLuanZZ** [comment]
> Please upload the picture correctly

![Screenshot_20240116-114705](https://github.com/ReVanced/revanced-manager/assets/108953018/44cc1d0d-e380-4d1f-8ae0-19406c900e36)

**Comment by oSumAtrIX** [comment]
What about sources?

**Comment by LLuanZZ** [comment]
> What about sources?

![Screenshot_20240116-115357](https://github.com/ReVanced/revanced-manager/assets/108953018/d88e3f3f-46f1-426e-b8d9-4f8f64b57e68)

**Comment by oSumAtrIX** [comment]
What if you click on the reset icon?

**Comment by LLuanZZ** [comment]
> What if you click on the reset icon? Check if this resolved the issue.

It worked, thank you very much bro.

**Comment by LLuanZZ** [comment]
> What if you click on the reset icon? Check if this resolved the issue.

Now another problem is happening, when I choose the YouTube APK, it goes back to Revanced and nothing happens.

**Comment by oSumAtrIX** [comment]
Open a new issue

**Comment by ILoveOpenSourceApplications** [comment]
It looks like you're using a VPN or Adblock of some sort. If so, try turning it off. Just a suggestion.

**Comment by JustCryen** [comment]
I've had the same issue. I just thought the development went quiet for a while…  
Resetting the sources and api url fixed this for me, thanks!  

I'm also using AdAway adblock but this doesn't seem to break anything as disabling it didn't change the behavior.  
Only after using the reset icons for both options the app came back to life (with AdAway enabled).
